test/jest/rendering-utils.js

```diff
@@ -0,0 +1,6 @@
+import { render } from "@testing-library/react";
+import * as React from "react";
+
+export const renderInSvg = (component) => {
```

Comment:
Now that we're not using enzyme's shallow renderer, we needed a way to prevent warnings about unrecognized SVG elements in the DOM. Rendering SVG elements in an SVG container fixes this problem.

Reply:
THANK YOU!  I was stubbing `console.error` for a while just to silence these errors; glad you've got a solution!